colab-notebooks-examples/search/04-multilingual.ipynb

```diff
@@ -0,0 +1,487 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "s49gpkvZ7q53"
+   },
+   "source": [
+    "# Multilingual search\n",
+    "\n",
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/elastic/elasticsearch-labs/main/blob/colab-notebooks-examples/search/04-multilingual.ipynb)\n",
+    "\n",
+    "In this example we'll use a multilingual embedding model\n",
+    "[multilingual-e5-base](https://huggingface.co/intfloat/multilingual-e5-base) to perform search on a toy dataset of mixed\n",
+    "language documents. Using this model, we can search in two ways:\n",
+    " * Across languages, for example using a query in German to find documents in English\n",
+    " * Within a non-English language, for example using a query in German to find documents in German\n",
+    "\n",
+    " While this example is using dense retrieval only, it's possible to also combine dense and tradition lexical retrieval\n",
+    " with hybrid search. For more information on lexical multilingual search, please see the blog post\n",
+    " [Multilingual search using language identification in Elasticsearch](https://www.elastic.co/blog/multilingual-search-using-language-identification-in-elasticsearch).\n",
+    "\n",
+    " The dataset used contains snippets of Wikipedia passages from the [MIRACL](https://project-miracl.github.io/) dataset."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "Y01AXpELkygt"
+   },
+   "source": [
```

Comment:
we can drop these requirements here. We have done for most recent notebook updates + only calling out ML node needs for when model is used in stack